src/workerd/io/worker-entrypoint.c++

```diff
@@ -111,6 +113,8 @@ class WorkerEntrypoint final: public WorkerInterface {
       kj::Date scheduledTime,
       uint32_t retryCount);
 
+  const kj::MonotonicClock& monotonicClock = kj::systemPreciseMonotonicClock();
```

Comment:
Minor, but it doesn't seem worth wasting a member variable on this, it is used no more than once per instance.

Reply:
:+1: I'll just get the clock when instantiating the durationExceededLambda then.